apps/api/src/employee/employee.service.ts

```diff
@@ -43,24 +51,83 @@ export class EmployeeService {
     return this.ienUsersRepository.findOne({ email });
   }
 
+  _nameSearchQuery(keyword: string) {
+    let keywords = keyword.split(' ');
+    keywords = keywords.filter(item => item.length);
+    if (keywords.length === 1) {
+      return `(Employee.name ilike '%${keywords[0].toLowerCase()}%')`;
```

Comment:
Isn't `toLowerCase` unnecessary when using `iLike`?

Reply:
quick search, it seems so, I'll look more into for final PR